### ISSUE TITLE
mmm-indent-line: Symbol’s function definition is void: nil

### ISSUE THREAD

**Issue by zweifisch** [AdamNiederer/vue-mode#65]
emacs 25.1.1
mmm-mode 20171004.336
vue-mode 20171013.1728

emacs complains when I open new line in pug template, pug is the new jade

**Comment by AdamNiederer** [comment]
Hi,

Could you try this with the new version of vue-mode (melpa should have it available in a few hours)? I haven't looked into this issue directly, but I've made a few changes to how vue-mode indents things which might fix this.

**Comment by Dogfalo** [comment]
Just updated today and now the error has changed slightly
```
vue-mmm-indent-line-narrowed: Symbol’s function definition is void: mmm-narrow-to-submode-region
```

**Comment by AdamNiederer** [comment]
I believe these issues were resolved in a34f8a14fe6a680bb18cfa4519f7984d8071a4e3. Feel free to re-open if anything else crops up.
